keyboards/planck/keymaps/jscotto/config.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
```

Comment:
Please remove the keymap files.  

a PR should be keymap changes, keyboard changes, or core changes, not a combination of them.  This keeps things more modular, and less "messy" if a PR needs to be reverted.